chart/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: llama-stack
```

Comment:
The name in the Chart.yaml file should match the folder name for the chart.

Mismatching these can cause import errors when publishing/using the chart.

My recommendation would be to update the folder structure to:

```
charts
└ llama-stack
   └ Chart.yaml
```